src/lumigo_tracer/wrappers/redis/redis_wrapper.py

```diff
@@ -66,7 +69,7 @@ def execute_wrapper(func, instance, args, kwargs):
     with lumigo_safe_execute("redis start"):
         commands = instance.command_stack
         command = [cmd[0] for cmd in commands if cmd] or None
```

Comment:
Also here, the value here might be both list or None, is this ok?

Reply:
here yes :-)